R/controls.R

```diff
@@ -39,9 +39,8 @@ filter_select <- function(id, label, sharedData, group, allLevels = FALSE,
     withKey = TRUE
   )
 
-  df <- df %>%
-    dplyr::group_by_(group = group) %>%
-    dplyr::summarise(key = list(key_))
+  # TODO: should this package really depend on dplyr?
+  df <- dplyr::summarise(dplyr::group_by(df, g = group), key = list(key_))
 
```

Comment:
@jcheng5 I'm sure I understand the intention here. Are you thinking there could/should be multiple groups for a given `sharedData` object?